### PR TITLE
Fix getting primary space index

### LINKED ISSUE #62: Select by space primary index fail
Call select with conditions by primary index returns error:

`SelectError: Failed to plan select: SelectPlanError: Passed bad conditions: ValidateConditionsError: No field or index "primary" found`

### PR DESCRIPTION
**Pull request by savolgin**
Closes #62

### CODE REVIEW

File: test/entrypoint/srv_select.lua [hunk 1]
```diff
@@ -24,7 +24,7 @@ package.preload['customers-storage'] = function()
                 if_not_exists = true,
                 engine = engine,
             })
-            customers_space:create_index('id', {
+            customers_space:create_index('id_index', {
```
Comment (olegrok) [inline test/entrypoint/srv_select.lua:27]: But seems we still need to test a case when field and index have the same name.

Comment (savolgin) [reply test/entrypoint/srv_select.lua:28]: Added indexes with same names as fields.

Comment (dokshina) [reply test/entrypoint/srv_select.lua:28]: Maybe it would be better to add unit tests for `get_index_by_name`? To make select tests more e2e?

Comment (savolgin) [reply test/entrypoint/srv_select.lua:28]: Test added

**Review by olegrok — APPROVED** [review]

**Comment by dokshina** [comment]
Thanks for your patch!

Please, update CHANGELOG before merging.

**Review by dokshina — APPROVED** [review]

**Comment by dokshina** [comment]
Thanks for your patch!

Please, update CHANGELOG before merging.

**Comment by savolgin** [comment]
Change log updated

**Pull request merged**